README.md

```diff
@@ -9,3 +9,8 @@ See [./docs/development-deployment.md](./docs/development-deployment.md)
 - [Blog](./docs/blog.md)
 - [People](./docs/people.md)
 - [Track Record Sections](./docs/track-record-sections.md)
+
+## Firebase links:
+- Production firebase console: [here](https://console.firebase.google.com/u/0/project/dvp-landing-266011/overview)
+- Staging firebase console: [here](https://console.firebase.google.com/u/0/project/dvp-landing-staging/overview)
+- Staging2 firebase console: [here](https://console.firebase.google.com/u/0/project/dvp-landing-static/overview)
```

Comment:
Nit: I these could remove the `u/0/` from the URLs then maybe it won't force a certain user Google Account. (Though I guess the default is going to be user 0 so :/)

Reply:
https://github.com/datavaluepeople/dvp-landing-static/pull/84/commits/80caa8a32a4d0f1e835f70cb07de9475edddabcc